### PR TITLE
Change groupId to com.iabtechlab to be able to distinguish Google and IAB artifacts

### PR DESCRIPTION
**Pull request by goldun**

### CODE REVIEW

File: openrtb-core/pom.xml [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
   <artifactId>openrtb-core</artifactId>
   <packaging>jar</packaging>
-  <name>Google OpenRTB Core</name>
+  <name>IAB OpenRTB Core</name>
 
   <parent>
-    <groupId>com.google.openrtb</groupId>
+    <groupId>com.iabtechlab</groupId>
```
Comment (simontrasler) [inline openrtb-core/pom.xml]: I would expect this (and the other below) needs to be `com.iabtechlab.openrtb`. If not, why not?

Comment (goldun) [reply openrtb-core/pom.xml]: Good point!

Comment (goldun) [reply openrtb-core/pom.xml]: Fixed

**Review by simontrasler — APPROVED** [review]
LGTM.

**Pull request merged**